srv/web/api/python.template

```diff
@@ -78,33 +81,49 @@ class View(object):
 	def __str__(self):
 		return json.dumps(self.__dict__)
 
-class RPCClient:
-	def __init__(self, connection):
-		self.connection = connection
+steamconn = None
+
+def connect(host, port, username, passw, verify_ssl=True):
```

Comment:
Thinking about it more - Do we need global `steamconn` here? In fact `connect` can return always a new RPCClient.